package.json

```diff
@@ -56,7 +56,7 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "better-sqlite3": ">=7.6.0"
+    "better-sqlite3": ">=9.3.0"
```

Comment:
generally okay with this change but we have to keep in mind that if we incorporate this as part of the mobile app, we want to be able to dedupe node-deps containing native pieces as much as possible with core next. given that, we usually had this set to something that was compatible with core (or else we'd have to make sure the version used in core is compatible with this one)

Reply:
Good thought. Downgraded in 60cca7536a252dda0d1361d6921c4c4b911a38d4.